package.json

```diff
@@ -31,21 +33,18 @@
     "prettier": "^2.0.4",
     "pretty-quick": "^2.0.1",
     "serve": "^11.2.0",
+    "ts-config-single-spa": "^1.7.0",
+    "typescript": "^3.8.3",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.9.0",
-    "webpack-merge": "^4.2.2",
-    "@babel/preset-typescript": "^7.9.0",
-    "eslint-config-ts-important-stuff": "^1.0.0",
-    "typescript": "^3.8.3",
     "webpack-config-single-spa-ts": "^1.4.2",
-    "ts-config-single-spa": "^1.7.0"
+    "webpack-dev-server": "^3.9.0",
+    "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "single-spa": "^5.3.1",
     "@types/jest": "^25.2.1",
     "@types/systemjs": "^6.1.0",
     "@types/webpack-env": "^1.15.1",
-    "single-spa-layout": "1.1.5"
```

Comment:
I've removed single-spa-layout since it and the registerApplication API should not be used together.